packages/core/src/modules/resume/resume-review.ts

```diff
@@ -0,0 +1,53 @@
+import { db } from '@oyster/db';
+
+import { ResumeReviewBullJob } from '@/infrastructure/bull/bull.types';
+import { registerWorker } from '@/infrastructure/bull/use-cases/register-worker';
+import { sendEphemeralSlackNotification } from '@/modules/notification/use-cases/send-ephemeral-slack-notification';
+import { ENV } from '@/shared/env';
+
+const targetChannelId = ENV.CAREER_RESUME_REVIEW_CHANNEL_ID;
+
+export const resumeReviewWorker = registerWorker(
+  'resume_review',
+  ResumeReviewBullJob,
+  async (job) => {
+    const { channelId, messageId, text, userId, threadId } = job.data;
+
+    // Check if the message is in the #career-resume-review channel
+    if (channelId !== targetChannelId) {
+      return;
+    }
+
+    // Edge case: Empty message
+    if (!text) {
```

Comment:
won't the Zod validator already have caught this edge case?

Reply:
Yup, removed this redundant edge case